src/components/30-textarea/textarea.hbs

```diff
@@ -0,0 +1,5 @@
+<label for="demo-{{name}}">Textarea</label>
```

Comment:
I'd like to see this same template logic applied to the `text-input.hbs` template (see previous comment).